scripts/translate_ciphers.py

```diff
@@ -36,10 +36,6 @@ def test_translate_all_cipher_names(self):
              "+ECDHE-ECDSA:+AES-128-GCM:+AEAD",
              "ECDHE-ECDSA-AES128-GCM-SHA256",
              "TLS-ECDHE-ECDSA-WITH-AES-128-GCM-SHA256"),
-            ("TLS_RSA_WITH_AES_256_CBC_SHA",
```

Comment:
This script is used in mbedtls-3.6 thus we should probably not do that. We have already done something like that in https://github.com/Mbed-TLS/mbedtls-framework/pull/127 though. We should probably instead revert the commit 97d7e2adfa2c05292edc7804fb0295ceb11992b0.